### DISCUSSION TITLE
guide/install/reverse-proxy

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#289] [Announcements]
# guide/install/reverse-proxy

The program listens to port 5244 by default. If you do reverse proxy, recommended you set siteurl (../../config/configuration.md#siteurl) so that alist can works fine. If you wa...

https://alist.nn.ci/guide/install/reverse-proxy.html

**Comment by Faramitas** [comment]
ngnix proxy manager的反代 在location中设置上面的还是无法让传递非标准端口，需要https://github.com/alist-org/alist/issues/726#issuecomment-1661627242 这样设置

**Reply by Faramitas** [reply]
proxy_set_header X-Forwarded-Host $http_host;
